Lib9c/Action/HackAndSlash.cs

```diff
@@ -186,6 +168,25 @@ public override IAccountStateDelta Execute(IActionContext context)
             sw.Stop();
             Log.Verbose("{AddressesHex}HAS use ActionPoint: {Elapsed}", addressesHex, sw.Elapsed);
 
+            // Validate about avatar state.
```

Comment:
```suggestion
            sw.Restart();
            // Validate about avatar state.
```

Reply:
This part is being processed within the method. 😅